### ISSUE TITLE
[Marketplace] Can't check out as newly invited shop owner

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2669]
### Expected behavior
Not sure if a new shop owner is supposed to configure shipping or not?

### Actual Behavior
When attempting to check out I get the "Shipping Method Not Configured" and clicking on the configure links takes me to a corrupted settings panel
![cart_checkout](https://user-images.githubusercontent.com/72819/29254992-c5d70be6-80cf-11e7-82a6-8dd4be733c22.png)

### Steps to Reproduce the Behavior
1. invite a new shop owner
1. As shop owner attempt to configure shipping

### Versions
```
Node: 7.10.0
NPM: 5.0.3
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.0-ce
```

**Comment by spencern** [comment]
This is by design right now as my current understanding is that merchants should not be configuring their own shipping charges, but if that understanding is wrong then we can add the shipping package to the default merchant packages array. This can be configured per marketplace as it's stored in the database, but currently there is no UI for managing it.

**Comment by zenweasel** [comment]
@spencern Sure but a) I don't get any shipping and b) that's probably not the UI that should show? Should I change the issue then to be "Correct shipping method not used when checking out" or something similar to reflect the actual problem?

**Comment by zenweasel** [comment]
Closed via #2741
